src/backend/aspen/database/models/pathogens.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from sqlalchemy import Column, ForeignKey, Integer, String, UniqueConstraint
+from sqlalchemy.orm import relationship
+
+from aspen.database.models.base import idbase
+
+
+class Pathogen(idbase):  # type: ignore
+    """Bacterial or Viral Pathogen Data"""
+
+    __tablename__ = "pathogens"
+
+    slug = Column(
+        String,
+        nullable=False,
+        unique=True,
+        comment=(
+            "Used as a URL param for differentiating functionality within CZGE, ex: SC2"
+        ),
+    )
+    name = Column(
+        String,
+        nullable=False,
+        unique=True,
+        comment=("full pathogen abbreviated name, ex: SARS-CoV-2"),
+    )
+
+
+class PathogenPrefix(idbase):  # type: ignore
```

Comment:
Totally my bad -- can we rename this table to PathogenRepoConfig or something? I think we may need to store more settings for pathogen <-> repo relationships than just a prefix.

Reply:
Updated diagram: https://lucid.app/lucidchart/218d1f7f-a16e-47c9-a95e-3a044b5750e2/edit?beaconFlowId=CAE35111EE2EE04E&invitationId=inv_cb8b25ba-5fc6-44f7-858e-230bf7408603&page=0_0#

Reply:
no problem! i'll re-request review when i've finished adding tht